rule_packages/c/Types.json

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
Delete this file, regenerate the `BitfieldTypes` package, and delete the excess tests.